src/main/java/org/launchcode/liftoffrecipetracker/controllers/CategoryController.java

```diff
@@ -0,0 +1,50 @@
+package org.launchcode.liftoffrecipetracker.controllers;
+
+import org.launchcode.liftoffrecipetracker.data.CategoryRepository;
+import org.launchcode.liftoffrecipetracker.models.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("category")
+public class CategoryController {
+
+@Autowired
+private CategoryRepository categoryRepository;
+
+    @GetMapping
+    public String displayAllCategories(Model model) {
+        model.addAttribute("title", "All Categories");
+        model.addAttribute("category", categoryRepository.findAll());
```

Comment:
Same deal here as above. Because this is relating to multiple categories it might be best to change it to categories for ease of understanding as we work through the project--especially if we have to come back to this file later and have to reorient to it quickly

Reply:
done!